<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting_Scene</class>
 <widget class="QWidget" name="Setting_Scene">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="b6_exit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>201</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QSlider" name="s1_dffectVolum">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>200</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="s2_musicVolum">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>320</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="l1_effect">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>79</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: normal normal 15px &quot;微软雅黑&quot;;
color: white;</string>
   </property>
   <property name="text">
    <string>音效控制</string>
   </property>
  </widget>
  <widget class="QLabel" name="l2_music">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>280</y>
     <width>79</width>
     <height>89</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: normal normal 15px &quot;微软雅黑&quot;;
color: white;</string>
   </property>
   <property name="text">
    <string>音乐控制</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>359</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="b2_animation">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string>闲置动画：开</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="b3_shake">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string>屏幕抖动：开</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
